.nvmrc

```diff
@@ -0,0 +1 @@
+16.10.0
```

Comment:
16 is quite a bit past LTS, is 18 compat. with these changes? (And what node version are the gh-actions runner images running on?)

Reply:
It /might/ be, but copying Playbooks for now since I know "that builds", and the corresponding version of webapp is so old. I'd like to get up to speed with @cpoile's work in Calls at some point too for a later webapp.